codes/quantum/qubits/xzzx.yml

```diff
@@ -0,0 +1,31 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id: xzzx
+physical: qubits
+logical: qubits
+
+name: 'XZZX surface code'
+introduced: '\cite{arXiv:2009.07851}'
+
+description: 'A family of stabilizer codes whose generators are \(XZXZ\) Pauli strings associated, clock-wise, to the vertices of each face of a two-dimensional lattice (with a qubit located at each vertex of the tessellation)'
+
+protection: 'As stabilizer \([[n=O(d^2), k=O(1), d]]\).'
+
+features:
+  threshold:
+  - '\(50\%\) with pure \(X, Y\) or \(Z\) errors.'
+  - '\(18.7\%\) at standard depolarising noise.'
```

Comment:
Not nearly detailed enough as there many results related to the type of decoder used.  It is also of great interest that the optimal decoder beats the single-letter channel capacity (Hashing bound) at large, but finite, bias.

Reply:
updated